Add Firebase App Distribution's integration documentation (#41)
xmartlabs/android-snapshot-publisher: README.md

    releaseNotes {
        // Release notes customization
    }
    firebaseAppDistribution {
        // Firebase App Distribution setup
    }
    googlePlay {
        // Google Play setup
    }
}
```
The [version](#version-customization) and [releaseNotes](#release-notes) blocks are used to perform the preparation process.

> Whereas [firebaseAppDistribution](#firebase-app-distribution) and [googlePlay](#google-play) blocks are used to perform the distribution process.

Reviewer: ```suggestion
[firebaseAppDistribution](#firebase-app-distribution) and [googlePlay](#google-play) blocks are used to perform the distribution process.
```

You can define it using a relative path (where the start point is the Android application module folder) or an absolute path.

### Firebase App Distribution
This block defines the configuration needed to deploy the artifacts in Firebase App Distribution's system.
This plugin uses [Firebase's App Distribution plugin](https://firebase.google.com/docs/app-distribution/android/distribute-gradle).

```groovy
snapshotPublisher {
    firebaseAppDistribution {
        appId = null
        distributionEmails = ""
        distributionGroupAliases = ""
        serviceAccountCredentials = file("your-key.json")
    }
    // ...
}
```
The only required field is `serviceAccountCredentials`.

> To release to Google Play you must create a service account with access to the Play Developer API.

Reviewer: ```suggestion
To release on Google Play you must create a service account with access to the Play Developer API.
```

Author: hahaha there's a bigger mistake
```suggestion
To release to Firebase you must create a Google service account with Firebase Quality Admin role.
```
